Headlines/src/Controllers/NewsPreviewViewController.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  NewsPreviewViewController.swift
+//  Canillitapp
+//
+//  Created by Ezequiel Becerra on 03/03/2018.
+//  Copyright © 2018 Ezequiel Becerra. All rights reserved.
+//
+
+import UIKit
+
+class NewsPreviewViewController: UIViewController {
+    var news: News?
+    weak var newsViewController: NewsTableViewController?
+
+    @IBOutlet weak var imageView: UIImageView!
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var subtitleLabel: UILabel!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        imageView.sd_setImage(with: news?.imageUrl)
+        titleLabel.text = news?.title
+        subtitleLabel.text = news?.source
+    }
+    
+    var previewActions: [UIPreviewActionItem] {
+        
+        let shareAction = UIPreviewAction(title: "Compartir Noticia", style: .default, handler: { (_, _) -> Void in
+            guard let n = self.news else {
+                return
+            }
+            
+            UIPasteboard.general.string = ShareCanillitapActivity.canillitappURL(fromNews: n)
+        })
+        
+        var actions = [shareAction]
+        
+        if newsViewController != nil {
+            let reactionsAction = UIPreviewAction(title: "Agregar Reacción", style: .default, handler: { (_, _) -> Void in
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/ba933f4049a44fb3a8fc655051f4e975dcf6eb8e/Headlines/src/Controllers/NewsPreviewViewController.swift#L40"></span>Line should be 120 characters or less: currently 122 characters</td>
    </tr>
  </tbody>
</table>